Feature/#45 detail fragment

onxmoreplz opened the pull request:
## ISSUE NUMBER : 45

resolved: API Data 파싱 및 렌더링

## 내용
- 상단 StatusBar 투명하게 수정
- DetailFragment Rating Bar 생성
- ResultFragment Item 레이아웃 수정
- 어플 아이콘 변경

onxmoreplz committed:
[Update] DetailFragment 레이아웃 수정

- Star Rating Dialogue 생성
- [더보기] 버튼 생성 및 펼치기 준비

onxmoreplz committed:
[Add] StatusBar 투명하게 만들기

onxmoreplz committed:
[Update] Detail Fragment 수정

onxmoreplz committed:
[Update] App Icon 이미지 파일 업로드 및 변경

onxmoreplz committed:
[Add] ResultFragment Item 버튼 추가

- 디자이너 분들이 추가하신 버튼(15분이내, 30분이내, ...)

onxmoreplz committed:
[Update] API Data 요청 코드 수정

- Response DTO Class 생성
- RecipeFinal DTO Class 생성

ashwon12 commented on app/src/main/java/com/example/myapplication/navigation/search/SearchFragment.kt:
```diff
@@ -233,18 +234,16 @@ class SearchFragment : Fragment() {
                 it.run {
-                    tempRandomRecipes.add(it)
+                    searchAdapter.randomRecipes = it.list!!
 
-                    searchAdapter.updateRandomRecipeList(tempRandomRecipes)
+                    searchAdapter.updateRandomRecipeList(searchAdapter.randomRecipes)
                     searchAdapter.notifyDataSetChanged()
```
도영님 혹시 fragment에서 data 다 안받고 바로 adapter로 넘긴 이유가 따로 있나요?!

psh2849 reviewed (approved):
잘 나옵니다! 수고하셨습니다~~

onxmoreplz committed:
[Update] SearchFragment, DetailFragment, ResultFragment 레이아웃 수정

- SwipeRefreshLayout 삭제
- SearchcFragment 리싸이클러뷰 Height 조정
- ResultFragment 리싸이클러뷰 Item 버튼 추가(잘 안됨)

onxmoreplz committed:
[Add] Spinner 제목 설정 기능 추가

- 필터 : "컷수", "시간" 설정 기능 추가
